docs/current_docs/cookbook/snippets/builds/cache/php/src/MyModule.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Directory;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class MyModule
+{
+    // Run a build with cache invalidation
+    #[DaggerFunction]
+    public function build(): string
+    {
+        return dag()
+            ->container()
+            ->from("alpine")
+            // comment out the line below to see the cached date output
+            ->withEnvVariable("CACHEBUSTER", date(DATE_RFC2822))
+            ->withExec(["date"])
+            ->stdout();
```

Comment:
```suggestion
            ->from('alpine')
            // comment out the line below to see the cached date output
            ->withEnvVariable('CACHEBUSTER', date(DATE_RFC2822))
            ->withExec(['date'])
            ->stdout();
```

Not necessary by any coding standard but in PHP:
[Single quotes](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.single) make it clear you are taking the string (almost) as is.

[Double quotes](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.double) allow for variable expansion and escape sequences.

i.e. 

```php
$date = '1970-01-01';

$singleQuotedString = 'Date: {$date}'; // 'Date: {$date}'

$doubleQuotedString = "Date: {$date}"; // 'Date: 1970-01-01'
```